### PR TITLE
Fixed broken link in documentation

### PR DESCRIPTION
**Pull request by slevental**

### CODE REVIEW

**Comment by MHBauer** [comment]
Hi. Thanks for the pull request.

I think you are trying to fix the link behavior in github. The docs are meant to be viewed when running `make docs` or when deployed at [docs.docker.com](https://docs.docker.com).

File: docs/extend/plugin_api.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Docker Engine.
 
 This page is intended for people who want to develop their own Docker plugin.
 If you just want to learn about or use Docker plugins, look
-[here](/extend/plugins).
+[here](plugins.md).
```
Comment (moxiegirl) [inline docs/extend/plugin_api.md]: @slevental Thank you for your contribution.  Can you test it using `make docs`? When I test this contribution it isn't fixing the problem.

This does:

```
 look [here](/extend/plugins).
```

**Comment by MHBauer** [comment]
Same comment for this as for #15465.

**Comment by moxiegirl** [comment]
Closing not a bug. See @MHBauer's comment.

**Pull request closed without merging**